pyserini/encoded_query_info.py

```diff
@@ -29,7 +39,27 @@
         "total_queries": 3610,
         "downloaded": False
     },
-    "dpr-trivia_qa-dev-multi": {
+    "dpr-nq-dev-ance-multi": {
```

Comment:
Related: https://github.com/castorini/pyserini/issues/396#issuecomment-789761717

i.e., consistent naming... but doesn't have to be in this PR.

Reply:
Let's fix in this PR.
will update soon

Reply:
fixed the encoded query by moving "dpr" to front